src/sendCommercialMetrics.ts

```diff
@@ -6,7 +6,7 @@ type CommercialMetrics = {
 	page_view_id: string;
 	platform: string;
 	metrics: Metrics[];
```

Comment:
If we’re there… we might as well:

```suggestion
	metrics: readonly Metrics[];
```

Reply:
Done